tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,81 @@
+"""Task 1 and 2 of Module 2."""
+
+from collections import defaultdict
+# create CONSTANT variables for weight
+LIGHT_WEIGHT_THRESHOLD = 60
+HEAVY_WEIGHT_THRESHOLD = 70
+MAX_INVENTORY_CAPACITY = 80
+
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+# update dragonLoot variable changed to dragon_loot
+dragon_loot = [
+
+    'gold coin', 'chewed gum', 'dagger', 'gold coin', 'gold coin', 'ruby',
+
+    'rubbish', 'chewed gum', 'used tissue']
+
+
+def display_inventory(inventory):
+    # change header for display_inventory function
+    """Check the inventory of a player and provides weight/speed dependency status."""
+    print("Inventory:")
```

Comment:
I'd keep this print statement directly over the for loop from line 24 to keep it nice and close to each other